wallet/test/de/schildbach/wallet/payments/SendCoinsTaskRunnerBIP70Test.kt

```
@@ -802,4 +811,217 @@ class SendCoinsTaskRunnerBIP70Test {
+        mockWebServer.enqueue(
+            MockResponse()
+                .setResponseCode(HttpURLConnection.HTTP_OK)
+                .setHeader("Content-Type", PaymentProtocol.MIMETYPE_PAYMENTACK)
+                .setBody(okio.Buffer().write(createPaymentAck("nack")))
+        )
+
+        val sendRequest = createTestSendRequest(testAddress, testAmount)
+        try {
+            sendCoinsTaskRunner.sendDirectPayment(sendRequest, paymentIntent)
+            fail("Expected DirectPayException for NACK")
+        } catch (e: org.dash.wallet.common.services.DirectPayException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)